dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/GridHeader.java

```diff
@@ -233,6 +256,18 @@ public LegendSet getLegendSetObject()
         return legendSet;
     }
 
+    @JsonProperty
+    public String getProgramStage()
+    {
+        return programStage != null && !programStage.isBlank() ? programStage : null;
```

Comment:
Instead use Apache `StringUtils.trimToNull` when setting the value, or when returning it. A bit strange to check if not null and if not then return null :-)

Reply:
Done